### PR TITLE
📖 Docs fixes for code blocks, links and minor typos

### PR DESCRIPTION
**Pull request by yuridekim**
I've recently come across some issues while reading the Kubebuilder document.

This PR mainly addresses:
- formatting issues that do not properly render code blocks
- broken links that reference the Cronjob code
- minor typos and grammar for a smoother reading experience

### CODE REVIEW

File: docs/book/src/multiversion-tutorial/api-changes.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ schedule: "*/1 * * * *"
 That's a pretty textbook example of a special string format (it's also
 pretty unreadable unless you're a Unix sysadmin).
 
-Let's make it a bit more structured.  According to the our [CronJob
-code][cronjob-sched-code], we support "standard" Cron format.
+Let's make it a bit more structured.  According to our [CronJob
+code][https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/multiversion-tutorial/testdata/project/api/v2/cronjob_types.go], we support "standard" Cron format.
```
Comment (camilamacedo86) [inline docs/book/src/multiversion-tutorial/api-changes.md:16]: We need to use alias as above [cronjob-sched-code]
Then, define the links at the bottom.

Could you please fix this one?
Also, did you check this change in the preview? 
It probably is not broken

File: docs/book/src/reference/webhook-overview.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Webhook
 
 Webhooks are requests for information sent in a blocking fashion. A web
-application implementing webhooks will send an HTTP request to other application
-when certain event happens.
+application implementing webhooks will send an HTTP request to other applications
```
Comment (varshaprasad96) [inline docs/book/src/reference/webhook-overview.md:4]: Nit:
```suggestion
application implementing webhooks will send a HTTP request to other applications
```

**Review by varshaprasad96 — COMMENTED** [review]
Looks good after @camilamacedo86's comment is addressed.

Thanks for working on this @yuridekim!
/lgtm

File: docs/book/src/reference/webhook-overview.md (hunk 2 above)
Comment (yuridekim) [reply docs/book/src/reference/webhook-overview.md]: Added!

File: docs/book/src/multiversion-tutorial/api-changes.md (hunk 1 above)
Comment (yuridekim) [reply docs/book/src/multiversion-tutorial/api-changes.md]: Didn't notice this thank you! I reverted it using the alias.

File: docs/book/src/multiversion-tutorial/api-changes.md [hunk 3]
```diff
@@ -52,4 +52,4 @@ Now, let's copy over our existing types, and make the change:
 
 Now that we've got our types in place, we'll need to set up conversion...
 
-[cronjob-sched-code]: /TODO.md
+[cronjob-sched-code]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/multiversion-tutorial/testdata/project/api/v2/cronjob_types.go "ConJob Code"
```
Comment (camilamacedo86) [inline docs/book/src/multiversion-tutorial/api-changes.md:55]: We must use relative PATHS?
Can you please fix this one?

Comment (yuridekim) [reply docs/book/src/multiversion-tutorial/api-changes.md]: Fixed in new commit

**Comment by Kavinjsir** [comment]
@yuridekim Would you squash the commits into one and rebase the branch based off the lastest master branch?

That should help fix the docs preview failure in CI.

**Comment by yuridekim** [comment]
> @yuridekim Would you squash the commits into one and rebase the branch based off the lastest master branch?
> 
> That should help fix the docs preview failure in CI.

Thanks, done the squashing and the rebasing. Ready for review.

**Comment by camilamacedo86** [comment]
/test pull-kubebuilder-e2e-k8s-1-25-3

that is great for me !!!!

🎉

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Pull request merged**